src/dealDashboard/discussionsService.ts

```diff
@@ -347,11 +356,14 @@ export class DiscussionsService {
         replyTo,
       );
 
+      if ((createResponse as any).error) throw (createResponse as any).error;
```

Comment:
Is typed as `IComment`, but the convoSdk also throws AbortController errors, so we catch it here